DEBUG:asyncio:Using selector: EpollSelector
DEBUG:tzlocal:/etc/timezone found, contents:
 Europe/Minsk

DEBUG:tzlocal:/etc/localtime found
DEBUG:tzlocal:2 found:
 {'/etc/timezone': 'Europe/Minsk', '/etc/localtime is a symlink to': 'Europe/Minsk'}
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "send_daily_message" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:aiogram.dispatcher:Start polling
DEBUG:apscheduler.scheduler:Looking for jobs to run
DEBUG:apscheduler.scheduler:Next wakeup is due at 2024-06-04 21:33:00+03:00 (in 84758.176979 seconds)
INFO:aiogram.dispatcher:Run polling for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.event:Update id=196020087 is not handled. Duration 0 ms by bot id=7291189937
ERROR:aiogram.event:Cause exception while process update id=196020087 by bot id=7291189937
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/middlwares.py", line 51, in __call__
    xx = 4 / 0
ZeroDivisionError: division by zero
WARNING:aiogram.dispatcher:Received SIGINT signal
INFO:aiogram.dispatcher:Polling stopped for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.dispatcher:Polling stopped
DEBUG:asyncio:Using selector: EpollSelector
DEBUG:tzlocal:/etc/timezone found, contents:
 Europe/Minsk

DEBUG:tzlocal:/etc/localtime found
DEBUG:tzlocal:2 found:
 {'/etc/timezone': 'Europe/Minsk', '/etc/localtime is a symlink to': 'Europe/Minsk'}
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "send_daily_message" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:aiogram.dispatcher:Start polling
DEBUG:apscheduler.scheduler:Looking for jobs to run
DEBUG:apscheduler.scheduler:Next wakeup is due at 2024-06-04 21:33:00+03:00 (in 84705.070346 seconds)
INFO:aiogram.dispatcher:Run polling for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.event:Update id=196020088 is not handled. Duration 0 ms by bot id=7291189937
ERROR:aiogram.event:Cause exception while process update id=196020088 by bot id=7291189937
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/middlwares.py", line 51, in __call__
    xx = 4 / 0
ZeroDivisionError: division by zero
INFO:aiogram.event:Update id=196020089 is not handled. Duration 0 ms by bot id=7291189937
ERROR:aiogram.event:Cause exception while process update id=196020089 by bot id=7291189937
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/middlwares.py", line 51, in __call__
    xx = 4 / 0
ZeroDivisionError: division by zero
WARNING:aiogram.dispatcher:Received SIGINT signal
INFO:aiogram.dispatcher:Polling stopped for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.dispatcher:Polling stopped
DEBUG:tzlocal:/etc/timezone found, contents:
 Europe/Minsk

DEBUG:tzlocal:/etc/localtime found
DEBUG:tzlocal:2 found:
 {'/etc/timezone': 'Europe/Minsk', '/etc/localtime is a symlink to': 'Europe/Minsk'}
INFO:apscheduler.scheduler:Adding job tentatively -- it will be properly scheduled when the scheduler starts
INFO:apscheduler.scheduler:Added job "send_daily_message" to job store "default"
INFO:apscheduler.scheduler:Scheduler started
INFO:aiogram.dispatcher:Start polling
DEBUG:apscheduler.scheduler:Looking for jobs to run
DEBUG:apscheduler.scheduler:Next wakeup is due at 2024-06-04 21:33:00+03:00 (in 84510.644355 seconds)
INFO:aiogram.dispatcher:Run polling for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.event:Update id=196020090 is not handled. Duration 0 ms by bot id=7291189937
ERROR:aiogram.event:Cause exception while process update id=196020090 by bot id=7291189937
ZeroDivisionError: division by zero
Traceback (most recent call last):
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/venv/lib/python3.8/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/home/onelifeitsme/PycharmProjects/tt_downloader/middlwares.py", line 50, in __call__
    xx = 4 / 0
ZeroDivisionError: division by zero
WARNING:aiogram.dispatcher:Received SIGINT signal
INFO:aiogram.dispatcher:Polling stopped for bot @tiktok_save_video_bot id=7291189937 - 'TikTok — скачать видео'
INFO:aiogram.dispatcher:Polling stopped
